classes/core-image-pelux.bbclass

```diff
@@ -33,9 +33,9 @@ IMAGE_INSTALL_append = "\
 "
 
 # Media playback daemon
-IMAGE_INSTALL_append = "\
-    mopidy \
-"
+#IMAGE_INSTALL_append = "\
+#    mopidy \
+#"
```

Comment:
In my opinion this should be removed, not commented out, and then the commit can be reverted once Mopidy is supported again.